### ISSUE TITLE
Integrity constraint violation

### ISSUE THREAD

**Issue by alex-brukhty** [putyourlightson/craft-blitz#699]
### Bug Report

Error
`SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '9-9' for key 'blitz_elementfieldcaches.PRIMARY' The SQL being executed was: INSERT INTO 'blitz_elementfieldcaches' ('cacheId', 'elementId',  ....`

It looks like it's relevant to [this issue](https://github.com/craftcms/cms/issues/15517)
Started after updates Craft 5.2.8 => 5.3.2 and blitz 5.4.0 => 5.6.2

### Diagnostics Report

Blitz: 5.6.2
CraftCMS: 5.3.3 and 5.3.2

**Comment by bencroker** [comment]
When are you seeing the error and what steps can I take to reproduce it? And have you tested on Craft 5.3.3 which, according to https://github.com/craftcms/cms/commit/32e3eb536422d17f991316994f59943baeac863d, fixes https://github.com/craftcms/cms/issues/15517?

**Comment by alex-brukhty** [comment]
Sorry @bencroker I should have explained better.
So after the update (Craft 5.2.8 => 5.3.2 and blitz 5.4.0 => 5.6.2) Sentry started getting these errors on every fresh page visit (means first visit before cahced). 
From logs looks like error is triggered by blitz plugin `2024-08-12 16:53:11 [web.ERROR] [blitz] SQLSTATE[23000]: Integ...`
The update to 5.3.3 didn't resolve the problem.

Tried: 
- `blitz/cache/flush`
- plugin uninstall/install
- roll back to 5.4.0

**Comment by bencroker** [comment]
I’m unable to reproduce using Craft 5.3.3 and Blitz 5.6.2. Are you sure all Blitz migrations were successfully run during the update to 5.6.2? And is there a stack trace with the error? That would help me track it down.

**Comment by alex-brukhty** [comment]
@bencroker hang on, looks like re-installing with `craft plugin/uninstall blitz` console helped.
Just generated 1000 pages, no errors logged.
Tested lockally and on the prod. Initally I tried re-installing with CP only.

I guess updading pluigin in CP didn't trigger migration, but my deployment script have `migrate/all` command, so not fully sure why it didn't happen. 

Sorry for false alarm.

**Comment by bencroker** [comment]
Strange, but glad to hear you figured it out!

**Comment by dletsche** [comment]
In case it's helpful, I just wanted to share that I ran into this same issue. An uninstall/reinstall of Blitz got rid of the errors.

In my case I uninstalled via the CP locally > deployed to remote server > reinstalled/reconfigured locally > confirmed no more errors > deployed to remote server > confirmed no more errors.

I first noticed the errors in the log yesterday while running Craft 5.3.3 and Blitz 5.6.2. 

I had updated from Craft 5.2.6 => 5.3.1 and Blitz 5.4.0 => 5.6.2 last week. Then Craft 5.3.3 yesterday. I looked back at logs prior to yesterday and didn't see the errors.

**Comment by bencroker** [comment]
@dletsche thanks for mentioning this. So am I correct in understanding that you started seeing errors in the logs only after updating to Craft 5.3.3?

**Comment by dletsche** [comment]
@bencroker - yep, correct. As best I can tell, the errors were introduced after updating to Craft 5.3.3 (from 5.3.1).

**Comment by bencroker** [comment]
Hmm, something sounds off to me. Can you share the full exception stack trace with me, or send to support@putyourlightson.com if it contains sensitive data, so I can look into it?

**Comment by john-henry** [comment]
I just had this issue also after upgrading Craft from 5.3.1 -> 5.3.4 and Blitz from 5.6.2 -> 5.6.4 at the same time.

Uninstalling locally, deploying and then following up with it installed fixed it for me

This was my deploy log

```
Checking for pending Craft and plugin migrations ...
Total 1 new Blitz migration to be applied:
    - m240815_120000_add_fieldinstanceuid_column

Skipping database backup.
*** applying m240815_120000_add_fieldinstanceuid_column
    > dropping {{%blitz_elementfieldcaches}} if it exists ... done (time: 0.011s)
    > create table {{%blitz_elementfieldcaches}} ... done (time: 0.005s)
    > dropping {{%blitz_elementqueryfields}} if it exists ... done (time: 0.004s)
    > create table {{%blitz_elementqueryfields}} ... done (time: 0.004s)
*** applied m240815_120000_add_fieldinstanceuid_column (time: 0.029s)


1 migration was applied.
```

Similar errors in my Rollbar logs

```
SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '324-91-dbec1501-1197-4a10-a157-7066f8774694' for key 'blitz_elementfieldcaches.PRIMARY'
The SQL being executed was: INSERT INTO `blitz_elementfieldcaches` (`cacheId`, `elementId`, `fieldInstanceUid`) VALUES (324, 91, 'dbec1501-1197-4a10-a157-7066f8774694'), (324, 1420, 'bb96174e-7a1c-44d4-8e29-ba468cf4458e'), (324, 1421, '743b993d-0782-4e4a-8c2b-cd652d852eea'), (324, 3564, 'bb96174e-7a1c-44d4-8e29-ba468cf4458e'), (324, 3565, '743b993d-0782-4e4a-8c2b-cd652d852eea'), (324, 3796, 'bb96174e-7a1c-44d4-8e29-ba468cf4458e'), (324, 3797, '743b993d-0782-4e4a-8c2b-cd652d852eea'), (324, 9238, 'bb96174e-7a1c-44d4-8e29-ba468cf4458e'), (324, 9239, '743b993d-0782-4e4a-8c2b-cd652d852eea'), (324, 16118, '36145e9e-ecb3-491e-843c-522018d6e1a6'), (324, 16118, ''), (324, 16654, 'bb96174e-7a1c-44d4-8e29-ba468cf4458e'), (324, 16674, '743b993d-0782-4e4a-8c2b-cd652d852eea'), (324, 17404, 'c1720979-f408-4c12-b99e-6075f3025048'), (324, 17404, '101eb34e-1a04-44e8-b16f-20f17d6ae72b'), (324, 17404, '8eb02985-2b13-4799-a001-a5d64aeeb8b7'), (324, 17404, '38556631-f97a-4904-a068-d2fd70c90781')
```

**Comment by bencroker** [comment]
Thanks, @john-henry, I’ve addressed this in https://github.com/putyourlightson/craft-blitz/commit/744bd6a7a9fa63c6c0122a87e58aa66b7f90ae24 for the next release.

**Comment by bencroker** [comment]
Released in [5.7.0](https://github.com/putyourlightson/craft-blitz/releases/tag/5.7.0).
